ouroboros-network-framework/src/Ouroboros/Network/Subscription/Dns.hs

```diff
@@ -237,14 +259,16 @@ dnsSubscriptionWorker'
     -> Tracer IO (WithDomainName DnsTrace)
     -> Tracer IO (WithAddr Socket.SockAddr ErrorPolicyTrace)
     -> NetworkMutableState Socket.SockAddr
-    -> Resolver IO
+    -> IO b
+    -> (b -> (Resolver IO -> IO (SubscriptionTarget IO Socket.SockAddr))
+          -> IO (SubscriptionTarget IO Socket.SockAddr))
     -> DnsSubscriptionParams a
     -> Main IO (PeerStates IO Socket.SockAddr) x
     -> (Socket.Socket -> IO a)
     -> IO x
 dnsSubscriptionWorker' snocket subTracer dnsTracer errorPolicyTracer
                        networkState@NetworkMutableState { nmsPeerStates }
-                       resolver
+                       setupResolver resolver
```

Comment:
where does `setupResolver` comes from? I just couldn't find it in this patch and the `dns` library.

Reply:
`setupResolver` is the name of an argument, (DNS.makeResolvSeed DNS.defaultResolvConf) in `dnsSubscriptionWorker'`.